[2025-07-13T11:08:33.797+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-07-13T11:08:33.819+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: employee_prediction_job.check_for_new_data manual__2025-07-13T11:08:32+00:00 [queued]>
[2025-07-13T11:08:33.828+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: employee_prediction_job.check_for_new_data manual__2025-07-13T11:08:32+00:00 [queued]>
[2025-07-13T11:08:33.829+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-07-13T11:08:33.845+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): check_for_new_data> on 2025-07-13 11:08:32+00:00
[2025-07-13T11:08:33.859+0000] {warnings.py:112} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=68) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-07-13T11:08:33.860+0000] {standard_task_runner.py:72} INFO - Started process 70 to run task
[2025-07-13T11:08:33.861+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'employee_prediction_job', 'check_for_new_data', 'manual__2025-07-13T11:08:32+00:00', '--job-id', '168', '--raw', '--subdir', 'DAGS_FOLDER/prediction_job.py', '--cfg-path', '/tmp/tmpvomre7a6']
[2025-07-13T11:08:33.863+0000] {standard_task_runner.py:105} INFO - Job 168: Subtask check_for_new_data
[2025-07-13T11:08:33.937+0000] {task_command.py:467} INFO - Running <TaskInstance: employee_prediction_job.check_for_new_data manual__2025-07-13T11:08:32+00:00 [running]> on host a0a54bfbbb9e
[2025-07-13T11:08:34.294+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='employee_prediction_job' AIRFLOW_CTX_TASK_ID='check_for_new_data' AIRFLOW_CTX_EXECUTION_DATE='2025-07-13T11:08:32+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-13T11:08:32+00:00'
[2025-07-13T11:08:34.295+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-07-13T11:08:34.308+0000] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2025-07-13T11:08:34.308+0000] {base.py:84} INFO - Retrieving connection 'processed_files_db'
[2025-07-13T11:08:34.317+0000] {logging_mixin.py:190} INFO - Found new files: ['employee_data2_with_errors_chunk_12.csv', 'employee_data2_with_errors_chunk_5.csv', 'test_airflow.csv', 'employee_data2_with_errors_chunk_14.csv', 'employee_data2_with_errors_chunk_19.csv', 'employee_data2_with_errors_chunk_4.csv', 'employee_data2_with_errors_chunk_11.csv', 'employee_data2_with_errors_chunk_16.csv', 'employee_data2_with_errors_chunk_17.csv', 'employee_data2_with_errors_chunk_3.csv', 'employee_data2_with_errors_chunk_1.csv', 'employee_data2_with_errors_chunk_9.csv', 'employee_data2_with_errors_chunk_18.csv', 'employee_data2_with_errors_chunk_8.csv', 'employee_data2_with_errors_chunk_20.csv', 'employee_data2_with_errors_chunk_7.csv', 'employee_data2_with_errors_chunk_13.csv', 'employee_data2_with_errors_chunk_10.csv', 'employee_data2_with_errors_chunk_2.csv']
[2025-07-13T11:08:34.318+0000] {python.py:240} INFO - Done. Returned value was: ['employee_data2_with_errors_chunk_12.csv', 'employee_data2_with_errors_chunk_5.csv', 'test_airflow.csv', 'employee_data2_with_errors_chunk_14.csv', 'employee_data2_with_errors_chunk_19.csv', 'employee_data2_with_errors_chunk_4.csv', 'employee_data2_with_errors_chunk_11.csv', 'employee_data2_with_errors_chunk_16.csv', 'employee_data2_with_errors_chunk_17.csv', 'employee_data2_with_errors_chunk_3.csv', 'employee_data2_with_errors_chunk_1.csv', 'employee_data2_with_errors_chunk_9.csv', 'employee_data2_with_errors_chunk_18.csv', 'employee_data2_with_errors_chunk_8.csv', 'employee_data2_with_errors_chunk_20.csv', 'employee_data2_with_errors_chunk_7.csv', 'employee_data2_with_errors_chunk_13.csv', 'employee_data2_with_errors_chunk_10.csv', 'employee_data2_with_errors_chunk_2.csv']
[2025-07-13T11:08:34.341+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-07-13T11:08:34.342+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=employee_prediction_job, task_id=check_for_new_data, run_id=manual__2025-07-13T11:08:32+00:00, execution_date=20250713T110832, start_date=20250713T110833, end_date=20250713T110834
[2025-07-13T11:08:34.397+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-07-13T11:08:34.426+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-07-13T11:08:34.428+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
